### ISSUE TITLE
Project should be packaged with npm

### ISSUE THREAD

**Issue by davejbax** [kalaakash/TestRepo#3]
It's 2020, and we all know that most, if not all, projects should be packaged with `npm`. This will allow us to install dependencies and have the project exist in the Node repositories if published, improving extensibility and reach.

**Comment by kalaakash** [comment]
I agree. Would you like to be added as a collaborator to work on this issue?
